src/LessonPreview/QuestionCard/QuestionCard.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import './QuestionCard.scss'
+
+const QuestionCard = ({ id, question, allAnswers, reading, deleteQuestion})=> {
+  let correctAnswer = allAnswers.find(answer => answer.correct === true).desc
+  let incorrectAnswers = allAnswers
+    .filter(answer => answer.correct === false)
+    .map(answer => <p>{answer.desc}</p>)
```

Comment:
delete functionality for a teacher to remove a question from the lesson before submitting